tests/test_warning.py

```diff
@@ -35,23 +34,22 @@ def test_currentphenomenons(requests_mock: Mock) -> None:
 
     current_phenomenoms = client.get_warning_current_phenomenoms(domain="32", depth=1)
 
-    assert type(current_phenomenoms.update_time) == int
-    assert type(current_phenomenoms.end_validity_time) == int
-    assert type(current_phenomenoms.domain_id) == str
+    assert isinstance(current_phenomenoms.update_time, int)
+    assert isinstance(current_phenomenoms.end_validity_time, int)
+    assert isinstance(current_phenomenoms.domain_id, str)
```

Comment:
Changing type to isinstance could be done everywhere in one separate PR, same for #679